src/sentry/api/endpoints/project_metrics_extraction_rules.py

```diff
@@ -0,0 +1,72 @@
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases import ProjectEndpoint
+from sentry.api.serializers import serialize
+from sentry.api.serializers.models.metrics_extraction_rules import MetricsExtractionRuleSerializer
+from sentry.models.project import Project
+from sentry.sentry_metrics.extraction_rules import (
+    delete_metrics_extraction_rules,
+    update_metrics_extraction_rules,
+)
+
+"""
+Open Questions
+1. Do we need to register the key metricsExtractionRules in the project config somehow?
+2. Do we need to prevent breaking something in the project config somehow?
```

Comment:
ideally, we would have validation provided by sentry_relay, similar to https://github.com/getsentry/sentry/blob/8bf93a27544d21d29e3bfc9487542f7533254950/src/sentry/relay/config/metric_extraction.py#L784